java/Cherepnin_Oleh/src/com/zoolatech/lecture4/tasks/_1/Container.java

```diff
@@ -0,0 +1,32 @@
+package com.zoolatech.lecture4.tasks._1;
+
+public class Container {
+    private String value;
+    private ContainerUpdatesListener innerContainer;
+
+    {
+        innerContainer = new ContainerUpdatesListener();
```

Comment:
don't use initialization blocks for instance fields initialization - you can do it directly in a place where variable is declared or in the constructor